app/src/authentication/authentication.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime, timedelta
+from typing import Union
+
+from fastapi.security import OAuth2PasswordBearer
+from jose import jwt
+from passlib.context import CryptContext
+from pydantic import BaseModel
+
+
+# In the deployed app to be passed as environemntal variables :D
+# This approach for testing only :)
+SECRET_KEY = "5cd4b99178b3ba30e00c33496599954825b365a4ade55bd37ecceabaffbfdee1"
+ALGORITHM = "HS256"
+ACCESS_TOKEN_EXPIRE_MINUTES = 15
```

Comment:
Usually, tokens should expire in an hour. Developers hate this type of auth because of short-living tokens and making that 15 minutes is making this problem worse.